modules/cache.py

```diff
@@ -0,0 +1,27 @@
+from collections import OrderedDict
+import logging
+
+from modules.metrics import MetricsHandler
+
+class MappingLRUCache:
+    def __init__(self, cacheSize):
+        self.dict = OrderedDict()
+        self.size = cacheSize
+
+    def find(self, alias):
+        if alias in self.dict:
+            self.dict.move_to_end(alias, last=False) #move alias to front of cache
+            logging.debug(f"alias: '{alias}' is grabbed from mapping")
+            MetricsHandler.cache_hits.inc()
+            return self.dict[alias]
+        MetricsHandler.cache_misses.inc()
+        return None
```

Comment:
if we invert the logic we can indent less, check it out 
```suggestion
        if alias not in self.dict:
            MetricsHandler.cache_misses.inc()
            return None

        self.dict.move_to_end(alias, last=False) #move alias to front of cache
        logging.debug(f"alias: '{alias}' is grabbed from mapping")
        MetricsHandler.cache_hits.inc()
        return self.dict.get(alias)
```